src/nvim/indent.c

```diff
@@ -396,11 +451,16 @@ int get_breakindent_win(win_T *wp, const char_u *line)
 
   // used cached indent, unless pointer or 'tabstop' changed
   if (prev_line != line || prev_ts != wp->w_buffer->b_p_ts
-      || prev_tick != buf_get_changedtick(wp->w_buffer)) {
+      || prev_tick != buf_get_changedtick(wp->w_buffer)
+      || prev_vts != wp->w_buffer->b_p_vts_array) {
     prev_line = line;
     prev_ts = wp->w_buffer->b_p_ts;
     prev_tick = buf_get_changedtick(wp->w_buffer);
-    prev_indent = get_indent_str(line, (int)wp->w_buffer->b_p_ts, wp->w_p_list);
+    prev_vts = wp->w_buffer->b_p_vts_array;
+    prev_indent = get_indent_str_vtab(line,
+                                      (int)wp->w_buffer->b_p_ts,
```

Comment:
2nd arg is `long`, not int. Why cast b_p_ts? Same issue with the other `get_indent_` function calls.

Reply:
I just never fixed that after turning args to long. There were `int`s in the original PR, hence we needed casts here.

Reply:
```c
int get_indent_str_vtab(const char_u *ptr, long ts, long *vts, bool list)
```

Reply:
Yep-yep, I fixed this call already!